Quadriga/src/main/webapp/WEB-INF/views/transformation/transformation.jsp

```diff
@@ -0,0 +1,223 @@
+		});
+		$("input[type=button]").button().click(function(event) {
+			event.preventDefault();
+		});
+		
+		$("#selectAllProjects").click(function(){
+			$(".projectList").prop('checked',$(this).prop('checked'));
+		});
+		$("#selectAllTransformations").click(function(){
+			$(".transformationList").prop('checked',$(this).prop('checked'));
+		});
+		
+	});
+	
+</script>
+<script type="text/javascript">
+$(document).ready(function () {
+    $('#confirmationTransformation').click(function () {
+    	if(jQuery('#divProjectList input[type=checkbox]:checked').length && jQuery('#headingTwo input[type=checkbox]:checked').length) {
+        	var projects = [];
+            $.each($("input[name='project']:checked"), function(){            
+                projects.push($(this).val());
+            });        
+    		var transformations = [];
+            $.each($("input[name='transformation']:checked"), function(){            
+                transformations.push($(this).val());
+            });
+        	$('#confirm').dialog({ modal: true,
+        						draggable:false,
+        						resizable:false,
+       							position:'center',
+       							width:'auto',
+       							height:'auto',
+       							title:"Confirm Transformation",
+        	   							
+       							open: function (type, data) {
+       						        $(this).parent().appendTo("form");
+       						    },
+       						    
+	       						 buttons: { "Submit": function() { $(this).dialog("close"); },"Cancel": function() { $(this).dialog("close"); } },
+       							
+				        		open: function(){           
+				        			var projects = [];
+				        	        $.each($("input[name='project']:checked"), function(){            
+				        	            projects.push($(this).val());
+				        	        });
+				        			
+				        			var transformations = [];
+				        	        $.each($("input[name='transformation']:checked"), function(){            
+				        	            transformations.push($(this).val());
+				        	        });
+    			
+				        			jQuery("#contentholder").html("<h3>Transformations:</h3> "+transformations.join("<br/>")+""+" <br/> <br/><h3>Projects </h3> " + projects.join("<br/>")+"");	
+				        		}  		
+				        	  }
+        					);   
+    					}
+		    	
+    				else{
+				 		$('#alert').dialog({ modal: true,
+						draggable:false,
+						resizable:false,
+						position:'center',
+						width:'auto',
+						height:'auto',
+						title:"Transformation Cannot be processed",
+									
+						open: function (type, data) {
+						    $(this).parent().appendTo("form");
+						    },
+						 buttons: { "Okay": function() { $(this).dialog("close"); } },
+		    				});	
+						jQuery("#alertholder").html("Please select atleast one transformation and project");
+		    			}
+    });    
+});
+</script>
+<script>
+<c:if test="${flash == 1}">
+	window.alert("Error fetching list of approved networks");
+</c:if>
+</script>
```

Comment:
remove this and use the flash message box that's already available in the skeleton (see above comment)